pytorch_lightning/callbacks/base.py

```diff
@@ -176,11 +176,21 @@ def on_epoch_end(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -
         pass
 
     def on_batch_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
-        """Called when the training batch begins."""
+        r"""
+        .. deprecated:: v1.6 This callback hook was deprecated in v1.6 and will be removed in v1.8. Use
+            ``on_train_batch_start`` instead.
```

Comment:
```suggestion
        .. deprecated:: v1.6 
            This callback hook was deprecated in v1.6 and will be removed in v1.8. Use
            ``on_train_batch_start`` instead.
```
this may need to start on a new line for sphinx to render. highly likely, not sure